packages/frontend/src/components/Tags/index.tsx

```diff
@@ -1,29 +1,52 @@
 import { Chip, Stack } from '@mui/material';
 import { useContext } from 'react';
+import { useTranslation } from 'react-i18next';
 
-import { SearchContext } from '../../contexts/SearchContext';
-import { formatYear, mapSortTypeToValue } from '../../utils/helpers';
+import { SearchContext, Sorting } from '../../contexts/SearchContext';
+import { formatYear } from '../../utils/helpers';
+import { EMimeType } from '../../constants/Data';
 
 export const Tags = () => {
+  const { t } = useTranslation(['search']);
   const { years, savedCheckboxes, sort } = useContext(SearchContext);
-  const fromYear = formatYear(years[0]);
-  const toYear = formatYear(years[1]);
-  let yearRange = '';
-  if (fromYear) {
-    yearRange += fromYear;
-    if (fromYear < toYear && toYear) {
-      yearRange += `- ${toYear}`;
+
+  const getYearTagName = (): string => {
+    const fromYear = formatYear(years[0]);
+    const toYear = formatYear(years[1]);
+    if (fromYear) {
+      return fromYear < toYear && toYear ? `${fromYear} - ${toYear}` : fromYear;
```

Comment:
Shouldn't we first check that is there any toYear before comparing it to fromYear?

Reply:
In frontend `toYear` calendar is disabled until `fromYear` is selected so it's reasonable to have it check bit afterwards

Reply:
I meant this bit `fromYear < toYear && toYear`

Reply:
Can `fromYear < toYear` be true if there is not `toYear`?

Reply:
No, indeed `toYear` check is redundant

Reply:
Do note that toYear is converted to 0 if it is null for that comparison. But presumably fromYear can't be less than 0, so this shouldn't be a problem.